frontend/src/components/ScoreContainer.tsx

```diff
@@ -37,10 +37,10 @@ const ScoreContainer: React.FC = () => {
     }, []);
 
     return isLoading ? (
-        <div>Loading your total score...</div>
+        <div>Caricamento del punteggio totale...</div>
     ) : totalScore !== null ? (
         <div className="score-container" style={{ paddingBottom: '20px' }}>
-            <p className="score-label" style={{ fontWeight: 'bold', fontSize: '18px' }}>Evaluation score:</p>
+            <p className="score-label" style={{ fontWeight: 'bold', fontSize: '18px' }}>Punteggio di valutazione:</p>
```

Comment:
I would put here `Valutazione finale:`, which means `Final score`

Reply:
The translation `Valutazione finale:` is in the wrong place, when the component is loading the text `Caricamento del punteggio totale...` is perfectly fine. `Valutazione finale:` has to replace the current text of the score label element.